bench-templates/src/macros/ec.rs

```diff
@@ -214,8 +214,9 @@ macro_rules! ec_bench {
                     let name = format!("{}::{}", $curve_name, stringify!($Group));
                     let mut rng = ark_std::test_rng();
 
-                    let g = <$Group>::rand(&mut rng).into_affine();
-                    let v: Vec<_> = (0..SAMPLES).map(|_| g).collect();
+                    let v: Vec<_> = (0..SAMPLES)
```

Comment:
Let's construct the projective points, and then use batch inversion to get the affine form. Otherwise the benchmark will be too slow just in the setup phase.

Reply:
Ping on this!